src/components/SearchGridFilter.vue

```diff
@@ -1,6 +1,17 @@
 <template>
-  <div :class="{ 'search-filters': true,
+<div>
+  <div class="is-hidden-desktop">
+    <transition name="modal">
```

Comment:
I don't think this transition is necessary. It's just appearing and disappearing. Can you remove it and their corresponding styles?

Reply:
Removed.